pkgs/unified_analytics/test/unified_analytics_test.dart

```diff
@@ -1013,7 +1013,7 @@ $initialToolName=${ConfigHandler.dateStamp},$toolsMessageVersion
       // Checks item 3
       // All tools must be under 36 characters (and enforce each tool
       // begins with a letter)
-      final RegExp toolLabelPattern = RegExp(r'^[a-zA-Z][a-zA-Z\_]{0,35}$');
+      final RegExp toolLabelPattern = RegExp(r'^[a-zA-Z][a-zA-Z\_-]{0,35}$');
```

Comment:
There are some restrictions in Google Analytics about sending some parameters in the body with only alphanumeric and underscores so I think it would be best to apply that for all items where we can

https://developers.google.com/analytics/devguides/collection/protocol/ga4/sending-events?client_type=gtag#limitations

Reply:
OK.  We will then have to have a separate field for the control flag and not use the same field for the control flag and the label.  I'll make that change.

Reply:
Chat offline indicates that the value sent to analytics can contain dashes.  So keeping the control flag and label as the same thing is fine.